src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public Figure getDefaultFigure() {
+        Figure circle = new Circle("white", 10);
+        return circle;
+    }
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor();
+        Figure [] figures = new Figure[5];
+        Random random = new Random();
+        int size = random.nextInt(90) + 10;
```

Comment:
The range for random sizes (90 and 10) should be defined as constants for better readability and maintainability. Consider using `private static final int MIN_SIZE = 10;` and `private static final int SIZE_RANGE = 90;`.